examples/tts/conf/talknet-spect.yaml

```diff
@@ -0,0 +1,237 @@
+name: &name TalkNetDurs
+sample_rate: &sr 22050
+
+train_dataset: ???
+validation_datasets: ???
+durs_path: ???
+
+model:
+  d_char: &d_char 128
+  n_mels: &n_mels 80
+  dropout: &dropout 0.1
+  separable: &separable true
+
+  train_ds:
+    dataset:
+      cls: "nemo.collections.asr.data.audio_to_text.AudioToCharWithDursDataset"
```

Comment:
Can you switch to the new hydra format that uses target and drop params? See https://github.com/NVIDIA/NeMo/blob/dev/examples/tts/conf/uniglow.yaml for an example